internal/generator/vector/conf_test.go

```diff
@@ -85,18 +85,25 @@ var _ = Describe("Testing Complete Config Generation", func() {
 				},
 			},
 			ExpectedConf: `
-# Logs from containers (including openshift containers)
+# Logs from containers
 [sources.raw_container_logs]
 type = "kubernetes_logs"
 glob_minimum_cooldown_ms = 15000
 auto_partial_merge = true
-exclude_paths_glob_patterns = ["/var/log/pods/openshift-logging_collector-*/*/*.log", "/var/log/pods/openshift-logging_elasticsearch-*/*/*.log", "/var/log/pods/openshift-logging_kibana-*/*/*.log", "/var/log/pods/*/*/*.gz", "/var/log/pods/*/*/*.tmp"]
+exclude_paths_glob_patterns = ["/var/log/pods/openshift-logging_*/*/*.log", "/var/log/pods/*/*/*.gz", "/var/log/pods/*/*/*.tmp"]
 pod_annotation_fields.pod_labels = "kubernetes.labels"
 pod_annotation_fields.pod_namespace = "kubernetes.namespace_name"
 pod_annotation_fields.pod_annotations = "kubernetes.annotations"
 pod_annotation_fields.pod_uid = "kubernetes.pod_id"
 pod_annotation_fields.pod_node_name = "hostname"
 
+# Logs from Event Router
+[sources.raw_eventrouter_log]
+type = "file"
```

Comment:
It a pod so it should still be kubernetes_logs

Reply:
Above we excluded all logs from openshift-logging namespace and kubernetes_log don't have param for path so we can't use it here.